sdk/keyvault/Azure.Security.KeyVault.Administration/src/Azure.Security.KeyVault.Administration.csproj

```diff
@@ -10,15 +10,24 @@
     <NoWarn>$(NoWarn);3021;CA1812</NoWarn>
   </PropertyGroup>
 
-  <Import Project="..\..\Azure.Security.KeyVault.Shared\src\Azure.Security.KeyVault.Shared.projitems" Label="Shared" />
+  <!-- <Import Project="..\..\Azure.Security.KeyVault.Shared\src\Azure.Security.KeyVault.Shared.projitems" Label="Shared" /> -->
```

Comment:
Nit: I'd just remove it. We can always add it later if needs be, but with it commented out it almost makes it seem like it has a future (I don't think it does...not until we, if ever, generate existing track 2 packages).

Reply:
Thanks, I meant to remove it!